client/app/dedicated/server/dedicated-server.controller.js

```diff
@@ -176,62 +169,6 @@ angular.module("App").controller("ServerCtrl", (NO_AUTORENEW_COUNTRIES, WEATHERM
         loadServer();
     });
 
-    $scope.editReverse = () => {
```

Comment:
It wasn't used ?

Reply:
Nope, replaced by `app.dedicated.server.update-reverse-dns` state